reactos/drivers/usb/usbhub/fdo.c

```diff
@@ -1404,8 +1436,15 @@ USBHUB_FdoQueryBusRelations(
 
     if (!DeviceRelations)
         return STATUS_INSUFFICIENT_RESOURCES;
-    DeviceRelations->Count = Children;
-    Children = 0;
+
+    DeviceRelations->Count = Children + ChildrenFromTop;
+    Children = ChildrenFromTop;
```

Comment:
It's not obvious why this is done, doing it right before the for loop would make it more clear in my eyes.